extensions/extensions-helm/aissemble-configuration-store-chart/templates/webhook-cleanup-job.yaml

```diff
@@ -0,0 +1,46 @@
+{{ if .Values.webhook.enable }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ index .Values "aissemble-quarkus-chart" "app" "name" }}-webhook-cleanup
+  annotations:
+    "helm.sh/hook": pre-delete, pre-upgrade
```

Comment:
Q/S: Does this regenerate the cert on upgrade, and if so, is that necessary? I'd think we'd want the cert to stay the same. Is the idea that they webhook values might have changed and affect cert generation?

Reply:
I just copied that hook from the spark-operator clean up job. However thinking it through now, I don't think it makes sense for it to run on `pre-upgrade` since we don't need to the certs to change.